src/frontend/hooks/useEstimatedUncompressedSize.ts

```diff
@@ -0,0 +1,11 @@
+export const useEstimatedUncompressedSize = (
+  platform: string,
+  compressedSize: number
+) => {
+  const baseEstimate = compressedSize * 2
+  const gapPercentage = platform === 'osx' ? 0.05 : 0.1
+
+  const gap = baseEstimate * gapPercentage
+
+  return baseEstimate + gap
+}
```

Comment:
what is the behavior if this over or underestimates the uncompressed size?

Reply:
is it used to calculate the % extraction? as in `sizeOfExtractedFiles / estimatedUncompressedSize * 100 = percentExtracted`? or is there another way we're getting the denominator?

Reply:
Does not calculate the extraction %, thats only at the BE which we receive through `window.api.onProgressUpdate`, just the installation model or game page `install size` and does not affect any other logic around: 

**Below is where is being implemented**

Installation Model
![image](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/75917402/08f6968e-32a5-48b2-b902-9a793a49e95d)

Game Page
![image](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/75917402/18828454-a7b2-478e-b35c-a35be9b1ea93)

Reply:
Also, I didn't include these tests cases in https://app.qase.io/project/HC?previewMode=side&suite=43, but I can include them. btw, this is the issue linked to this change: https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/issues/614